src/main/java/net/fabricmc/loom/api/LoomGradleExtensionAPI.java

```diff
@@ -130,6 +132,20 @@ default void interfaceInjection(Action<InterfaceInjectionExtensionAPI> action) {
 	 */
 	Property<Boolean> getEnableTransitiveAccessWideners();
 
+	@ApiStatus.Experimental
+	IntermediateMappingsProvider getIntermediateMappingsProvider();
+
+	@ApiStatus.Experimental
+	<T extends IntermediateMappingsProvider> void setIntermediateMappingsProvider(Class<T> clazz, Action<T> action);
```

Comment:
Why not allow directly setting the instance considering it's constructed immediately as far as I can see?

Reply:
Oh I see it uses that abstract `Property` pattern, didn't see that before